rmgpy/rmg/main.py

```diff
@@ -488,6 +489,11 @@ def initialize(self, **kwargs):
             for reactionSystem in self.reactionSystems:
                 reactionSystem.viscosity = solventData.getSolventViscosity(reactionSystem.T.value_si)
 
+        # Read custom reaction filtering YAML file here and store content as lists
+        if self.filterReactions:
+            KineticsDatabase.loadFilterFits(self.database.kinetics, self.databaseDirectory,
+                                            self.reactionSystems)
```

Comment:
Should this be `self.database.kinetics.loadFilterFits(self.databseDirectory, self.reactionSystems)`?